src/mon/LogMonitor.cc

```diff
@@ -356,16 +355,26 @@ void LogMonitor::log_external(const LogEntry& le)
     channel = CLOG_CHANNEL_CLUSTER;
   }
 
+  string level = channels.get_log_level(channel);
+  int log_level = LogEntry::str_to_level(level);
+  if (log_level > le.prio) {
+    // Do not log LogEntry to any external entity if le.prio is
+    // less than channel log level.
+    return;
```

Comment:
ACK. That's the central place of the change, the place where we discard unwanted log entries. I was surprised there is real user of `log_file_level` in the current code base and did following archaeological excavations:

1. the `log_file_level` has been introduced in ef91c699616eaf5d82a451a7983bdee5158402c4;
```
       if (g_conf->mon_cluster_log_file.length()) {
-       blog.append(s + "\n");
+       int min = string_to_syslog_level(g_conf->mon_cluster_log_file_level);
+       int l = clog_type_to_syslog_level(le.type);
+       if (l <= min) {
+         stringstream ss;
+         ss << le << "\n";
+         blog.append(ss.str());
+       }
       }
```
2. its fundamental logic has been kept intact by b97b06e17f96f79911c860fac71c35e874afca65,
3. till 7c84e06e6f846f6b4b6fd959218b4d474520f429 which has introduced `LogMonitor::log_external` but also removed the check.

Therefore I see the restoration aspect of this change.

Reply:
Thanks Radek.